contributors/triage/guideline.md

```diff
@@ -0,0 +1,60 @@
+# Kubernetes Issue Triage Guidelines
+
+## Purpose
+
+Speed up issue management.
+
```

Comment:
@spzala this has to do with the formatting of this doc.   Can you use textwrap at 80 chars or something like that so long lines are broken up.  That way we can comment, give feedback with more granularity.